openapi.yaml

```diff
@@ -8916,3 +9072,101 @@ components:
       - $ref: '#/components/schemas/DeploymentAnalysis'
       description: Everything we know about the deployment.
       type: object
+    WorkTypeName:
+      description: Work type name. It is unique within the account scope.
+      minLength: 3
+      type: string
+    WorkTypeGetRequest:
+      description: Identifier of a work type - a set of rules to group PRs, releases,
+        etc. together.
+      example:
+        account: 1
+        name: Bug Fixing
+      properties:
+        account:
+          description: User's account ID.
+          type: integer
+        name:
+          $ref: '#/components/schemas/WorkTypeName'
+      required:
+      - account
+      - name
+      type: object
+    WorkTypes:
+      description: |
+        List of work types. Response from `/settings/work_types/{account}`.
+      items:
+        $ref: '#/components/schemas/WorkType'
+      type: array
+    WorkType:
+      description: Definition of a work type - a set of rules to group PRs, releases,
+        etc. together.
+      example:
+        name: Bug Fixing
+        color: FF0000
+        rules:
+        - name: pull_request/label_include
```

Comment:
This rule name is a bit weird to me as we could filter pull requests by Jira labels as well.

Reply:
The API has very different semantics for filtering by JIRA. It would look like `pull_request/jira/label_include`. Anyway, this is just a meaningless example to demo the schema.

Reply:
Yes ofc, it was just a random thought to be sure you were aware rules can combined jira filtering on gh data.